app/code/Magento/Catalog/Api/Data/ProductRenderSearchResultsInterface.php

```diff
@@ -27,4 +27,35 @@ public function getItems();
      * @return $this
      */
     public function setItems(array $items);
+
```

Comment:
Hi @jigneshkhunt. Thanks for collaboration. Due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't change interfaces. You can create a new interface in case if you need it.

Reply:
Hi @VladimirZaets , Thanks for update. I can create a new interface, but after then I need to update the return type of getList method. is it fine to update return type?

Reply:
Hi @jigneshkhunt. Can you please explain why you will need to update the return type of getList method?